config/main.py

```diff
@@ -2001,6 +2001,63 @@ def shutdown(ctx, interface_name):
         else:
             config_db.mod_entry("PORTCHANNEL", interface_name, {"admin_status": "down"})
 
+#
```

Comment:
Usually when we add a new CLI command in SONiC we don't mix other existing commands with new options.
A correct implementation will be:
* config txmon set Ethernet# 10
* config txmon clear Ethernet#
* config txmon period 10
* config txmon state enable/disable